engine/gamesys/src/gamesys/components/comp_collision_object.h

```diff
@@ -104,15 +64,32 @@ namespace dmGameSystem
         dmPhysicsDDF::CollisionShape::Type m_Type;
     };
 
-    bool GetShapeIndex(void* _component, dmhash_t shape_name_hash, uint32_t* index_out);
-    bool GetShape(void* _world, void* _component, uint32_t shape_ix, ShapeInfo* shape_info);
-    bool SetShape(void* _world, void* _component, uint32_t shape_ix, ShapeInfo* shape_info);
-
-
-    // For script_box2d.cpp
-    b2World* CompCollisionObjectGetBox2DWorld(dmGameObject::HComponentWorld world);
-    b2Body* CompCollisionObjectGetBox2DBody(dmGameObject::HComponent component);
-    dmGameObject::HInstance CompCollisionObjectGetInstance(void* _user_data);
+    // Adapter API
```

Comment:
I think this is just for scripts, so maybe it should be in comp_collision_object_script or something?